src/api/entities/Portfolio/index.ts

```diff
@@ -217,4 +221,83 @@ export class Portfolio extends Entity<UniqueIdentifiers> {
       return owner;
     }
   }
+
+  /**
+   * Retrieve a list of transactions where this portfolio was involved. Can be filtered using parameters
+   *
+   * @param filters.account - account involved in the transaction
```

Comment:
It isn't clear what this parameter represents. Which account? The sender or the receiver? Do we even want to expose this @VictorVicente ?

Reply:
We should ask it to backend-team.

Reply:
@shuffledex change the description to "account involved in the settlement"